src/scripts/site.js

```diff
@@ -227,12 +227,6 @@ var AppInfo = {};
 !function () {
     "use strict";
 
-    function onApiClientCreated(e, newApiClient) {
-        if (window.$) {
```

Comment:
Any ideas why they were replacing the ajax method here?

Reply:
Absolutely no clue, but it ran totally fine after removing these sections so I think we're in the clear.